smarts/env/wrappers/standard_obs.py

```diff
@@ -0,0 +1,582 @@
+    
+    box:
+        Bounding box of neighbor vehicles. Defaults to np.array([0,0,0]) per 
+        vehicle. shape=(10,3). dtype=np.float32.
+    heading:
+        Heading of neighbor vehicles in radians [-pi, pi]. Defaults to 
+        np.array([0]) per vehicle. shape=(10,). dtype=np.float32.
+    lane_index:
+        Lane number of neighbor vehicles. Defaults to np.array([0]) per 
+        vehicle. shape=(10,). dtype=np.int8.
+    pos:
+        Coordinate of the center of neighbor vehicles' bounding box's bottom 
+        plane. Defaults to np.array([0,0,0]) per vehicle. shape=(10,3). 
+        dtype=np.float32.
+    speed:
+        Speed of neighbor vehicles in m/s. Defaults to np.array([0]) per
+        vehicle. shape=(10,). dtype=np.float32.
+    """
+    ogm: Optional[np.ndarray] = None
+    """Occupancy grid map. Map is binary, with 255 if a cell is occupied, else
+    0. dtype=np.uint8."""
+    rgb: Optional[np.ndarray] = None
+    """RGB image, from the top view, with ego vehicle at the center.
+    shape=(height, width, 3). dtype=np.uint8."""
+    ttc: Optional[Dict[str, np.ndarray]] = None
+    """Time and distance to collision. Enabled only if both `waypoints` and
+    `neighborhood_vehicles` attributes are enabled in AgentInterface.
+    
+    angle_error:
+        Angular error in radians [-pi, pi]. shape=(1,). dtype=np.float32.
+    distance_from_center:
+        Distance of vehicle from lane center in meters. shape=(1,). 
+        dtype=np.float32.
+    dtc:
+        Distance to collision on the right lane (`dtc[0]`), current lane 
+        (`dtc[1]`), and left lane (`dtc[2]`). If no lane is available, to the 
+        right or to the left, default value of 0 is padded. shape=(3,). 
+        dtype=np.float32.
+    ttc:
+        Time to collision on the right lane (`ttc[0]`), current lane
+        (`ttc[1]`), and left lane (`ttc[2]`). If no lane is available,
+        to the right or to the left, default value of 0 is padded. shape=(3,).
+        dtype=np.float32.
+    """
+    waypoints: Optional[Dict[str, np.ndarray]] = None
+    """Feature array of 20 waypoints ahead or in the mission route, from the 
+    nearest 4 lanes. If lanes or waypoints ahead are insufficient, default 
+    values are padded.
+    
+    heading:
+        Lane heading angle at a waypoint in radians [-pi, pi]. Defaults to
+        np.array([0]) per waypoint. shape=(4,20). dtype=np.float32.
+    lane_index:
+        Lane number at a waypoint. Defaults to np.array([0]) per waypoint.
+        shape=(4,20). dtype=np.int8.
+    lane_width:
+        Lane width at a waypoint in meters. Defaults to np.array([0]) per
+        waypoint. shape=(4,20). dtype=np.float32.
+    pos:
+        Coordinate of a waypoint. Defaults to np.array([0,0,0]). 
+        shape=(4,20,3). dtype=np.float32.
+    speed_limit:
+        Lane speed limit at a waypoint in m/s. shape=(4,20). dtype=np.float32.
+    """
+
+
+class StandardObs(gym.ObservationWrapper):
```

Comment:
Given the possible confusion between `StandardObs` and `StdObs` noted above, I would suggest renaming this class to `StdObsWrapper`.

Reply:
Answered above.